src/jackdaw/streams/interop.clj

```diff
@@ -396,6 +408,11 @@
                  ^KTable (ktable* other-ktable)
                  ^ValueJoiner (value-joiner value-joiner-fn))))
 
+  (suppress
+    [_ suppress]
+    (clj-ktable
+       (.suppress ktable (suppress-config suppress))))
```

Comment:
it looks like the spec defined before was named `suppress-config` but the fn that uses it is called `suppress-config` and the config is named `suppress`. this seems a little confusing to me. 

might be helpful to name the config arg `suppress-config` to match the spec and the fn `create-suppress-config` or something. the names i suggested could definitely be better though

Reply:
I've tried to make more sense of the names. The argument is now `suppress-config` and the function is `suppress-config->suppressed` which is what it's doing actually: turning a `suppress-config` into a `org.apache.kafka.streams.kstream.Suppressed`.